pkg/utils/kube/kube.go

```diff
@@ -404,6 +404,34 @@ func GetDeploymentReplicas(u *unstructured.Unstructured) *int64 {
 	return &val
 }
 
+func GetResourceImages(u *unstructured.Unstructured) []string {
```

Comment:
For [CronJobs](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#example) it seems that the image is under `spec.jobTemplate.spec.template.spec.containers`. Should we also check there?

Reply:
Done in 6b9063a72199550b4fd51beabdabdc3104b1f7fb